aten/src/ATen/native/native_functions.yaml

```diff
@@ -1930,7 +1930,11 @@
 
 - func: max.dim_max(Tensor self, int dim, bool keepdim=False, *, Tensor(a!) max, Tensor(b!) max_values) -> (Tensor(a!) values, Tensor(b!) indices)
 
-- func: max_values(Tensor self, int[1] dim, bool keepdim=False) -> Tensor
+- func: max_values(Tensor self, int[1] dim=[], bool keepdim=False) -> Tensor
```

Comment:
What is the difference between max_values and amax?

Reply:
There is no difference. I just don't know how to deal with it. Deprecate, remove, or expose it as well?

Reply:
I would deprecate it:


```
TORCH_WARN_ONCE("torch.max_values is deprecated and will be removed in a future PyTorch release. Use torch.amax, instead. This warning will appear only once per process."
```

Or similar? 

I would not increase its exposure, either.

Reply:
OK, that makes sense.